Makefile

```diff
@@ -41,7 +41,7 @@ define build_image
 		$(eval BUILD_ARGS := --build-arg BASE_IMAGE=$(BASE_IMAGE_NAME)),
 		$(eval BUILD_ARGS :=)
 	)
-	$(CONTAINER_ENGINE) build --no-cache  -t $(IMAGE_NAME) $(BUILD_ARGS) $(2)
```

Comment:
I need to either have this parametrized, so that some builds can be done with caching, or I need to run podman directly without using the Makefile.

Reply:
done, I undid the hardcoded parameters in Makefile